ups/tmpl.cfg

```diff
@@ -19,6 +19,6 @@ dependencies = {
 # For more information, see the sconsUtils Doxygen documentation.
 config = lsst.sconsUtils.Configuration(
     __file__,
-    headers=["lsst/TMPL.h"],
+    headers=["lsst/tmpl/ExampleOne.h"],
```

Comment:
Not `Two` and `Three` also?

Reply:
This list is only used if you explicitly tell SCons to verify that dependencies' headers can be found when trying to build the dependent product, which is functionality that has been ignored ever since it was made optional (and basically unnecessary) since the first version of sconsUtils.  So yes, to the extent it should exist at all, one header here is sufficient.